x-pack/plugins/security_solution/public/common/components/event_details/cti_details/helpers.tsx

```diff
@@ -81,7 +82,9 @@ export const getEnrichmentIdentifiers = (enrichment: CtiEnrichment): CtiEnrichme
   field: getEnrichmentValue(enrichment, MATCHED_FIELD),
   value: getEnrichmentValue(enrichment, MATCHED_ATOMIC),
   type: getEnrichmentValue(enrichment, MATCHED_TYPE),
-  provider: getShimmedIndicatorValue(enrichment, PROVIDER),
+  provider:
```

Comment:
I don't think we want to conflate these two values as they're different concepts. Feed name is the data feed containing the indicator, while the provider is the entity that discovered/generated the indicator.

Reply:
So, do you suggest that we need to keep only FEED_NAME here?

It's basically will be the same as old behaviour